events/api/README.md

```diff
@@ -0,0 +1,59 @@
+# Nessie Events API
+
+This module contains the API for the Nessie events notification system.
+
+## Overview
+
+The `org.projectnessie.events.api` package contains the API classes for the Nessie events
+notification system.
+
+The main entry point of the API is the `Event` interface. Instances of `Event` are created and
+published by the Nessie server and consumed by registered event subscribers.
+
+The `Event` interface has a `getType()` method that returns the type of the event. The type is an
+enum that uniquely identifies the event type. This enum serves as a discriminator for the
+event type and allows subscribers to filter events by type. It also allows subscribers to
+easily serialize and deserialize events to and from JSON.
+
+The following concrete event implementations are defined:
+
+* `CommitEvent` - This event is published when a commit is created.
+* `MergeEvent` - This event is published when a merge is created.
```

Comment:
nit: `when a merge is created`... maybe `when a merge is committed`? A merge is not an entity in Nessie... not sure about "creating a merge" :thinking:

Reply:
You are right, how about "performed"?

Reply:
`performed` SGTM :+1: